export_libraries.php

```diff
@@ -0,0 +1,172 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Export all libraries script.
+ *
+ * @package   mod_hvp
+ * @author    Rossco Hellmans <rosscohellmans@catalyst-au.net>
+ * @copyright Catalyst IT, 2021
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+require_once('../../config.php');
+require_once($CFG->libdir . '/adminlib.php');
+require_once('locallib.php');
+
+// No guest autologin.
+require_login(0, false);
```

Comment:
is this require login actually required? admin_externalpage_setup already does this for you

Reply:
You're right this isn't needed, have removed it